index.html

```diff
@@ -69,6 +69,18 @@ <h3 class="featured-project_head">
                 <a class="call-to-action" href="http://cfpb.github.io/api/ccdb/">API docs</a>
             </div>
         </li>
+        <li class="featured-project">
+            <div class="featured-project_img__ds"></div>
+            <h3 class="featured-project_head">
+                <a href="http://cfpb.github.io/design-manual/">Design System</a>
```

Comment:
This is linking to the design-manual (which redirects, but then we get a banner). 
It should be https://cfpb.github.io/design-system/